test/test.oauth2.ts

```diff
@@ -1338,6 +1369,27 @@ describe('oauth2', () => {
       assert.deepStrictEqual(info.scopes, tokenInfo.scope.split(' '));
     });
 
+    it('should refresh request header when refreshHandler is available', async () => {
```

Comment:
This test tests refresh handler flow when the existing token is expired.
Can you add a test when it is not expired? Also update the test description to clarify when a test is testing expiration flow or non-expiration flow.